apps/root-admin/src/app/(dashboard)/festivals/[id]/page.tsx

```diff
@@ -1,22 +1,28 @@
-import { StatusNotifier, TableBoard } from '@components/festival';
+import { StatusNotifier, ConcertBoard } from "@components/festival";
+import { Flex } from "antd";
+import { getDetailFestival } from "./data";
 import type { Params } from '@swifty/shared-lib';
 
-import { getMockFestivalDetail } from './data';
 import styles from './page.module.css';
 
 export default async function Page({ params: { id } }: Params<{ id: string }>) {
-  const { adminFestivalInfoResponse, adminConcertInfoResponses } =
-    await getMockFestivalDetail(id);
+  const {
+    adminFestivalInfoResponse,
+    adminConcertInfoResponses
```

Comment:
결국 저희가 사용하는 데이터는 각 페스티벌 정보, 콘서트 정보이고 Response는 API와 연관된 용어라고 생각합니다. 그래서 Response는 해당 데이터의 역할과 연관이 없다고 생각해서 아래와 같이 표현하는 건 어떨까요?
```suggestion
    adminFestivalInfo,
    adminConcertInfo
```